ffi/capi/src/custom_writeable.rs

```diff
@@ -116,3 +116,56 @@ pub unsafe extern "C" fn icu4x_simple_writeable(buf: *mut u8, buf_size: usize) -
         grow,
     }
 }
+
+
+#[no_mangle]
+pub unsafe extern "C" fn icu4x_buffer_writeable(cap: usize) -> *mut ICU4XWriteable {
```

Comment:
issue: Please add doc comments to these, highlighting any safety constraints and relations with other functions ("use this function to call destructors", etc)